src/openknotscore/pipeline/import_source.py

```diff
@@ -0,0 +1,96 @@
+import glob
+import itertools
+import pandas as pd
+import rdat_kit
+
```

Comment:
I think we should clarify that we expect the input files here be RDATs, with the expectation that we can support other options in the future if needed (for example, `load_csv`, `load_parquet`, `load_pickle`)